services/movie_session.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime.datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(
+        session_date: datetime.date = None
+) -> MovieSession:
```

Comment:
Incorrect type hint for return value. This function returns QuerySet of MovieSession